### PR TITLE
Feat/38 nav bar

### LINKED ISSUE #38: Navigation Bar
As a user, I need to be able to move around in the application with ease and a good understanding of where I am.  

### Functional Requirements
* Add a global navigation bar that will be rendered on top of every page, or where appropriate.  
* Navigation bar should display the user's current location.
* Nav bar should also allow the user to travel to different sections of the application.
* Nav bar needs to be a consistent theme with the one on the main landing page.

### Technical Requirements 
* Create a nav bar object in html that we can render where ever we need to.
* Good place for the nav_bar.html.erb to live is App_Views_Layouts. 
* Establish two independent nav bars that we can render depending on whether the user is logged in or not, this way we can easily have matching nav bars with different functionalities before the user logs in and after.

### PR DESCRIPTION
**Pull request by sganous**
Closes #38

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -2,6 +2,9 @@ class StaticPagesController < ApplicationController
   skip_before_action :ensure_signed_in
 
   def landing
+    if user_signed_in?
+      redirect_to company_job_postings_path(current_user[:company_id])
```
Comment (S1v4) [inline app/controllers/static_pages_controller.rb]: `current_user` is an object. Use `current_user.company_id` so we don't confuse it for a hash. Although `current_user[:company_id]` works, it's not meant to used simple instance variable accessing. Read the first method of: http://api.rubyonrails.org/classes/ActiveRecord/AttributeMethods.html.
```ruby
  def current_user
    @current_user ||= User.find current_user_id
  end
  helper_method :current_user
```

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -8,6 +8,13 @@
     <link href='https://fonts.googleapis.com/css?family=Sofia' rel='stylesheet'>
```
Comment (S1v4) [inline app/views/layouts/application.html.erb]: does the `limelight` brand not have `sofia` font without this line being in both partials?

Comment (S1v4) [reply app/views/layouts/application.html.erb]: I also want to move this into `application.css instead`

Comment (sganous) [reply app/views/layouts/application.html.erb]: It doesn't unfortunately :/

File: app/controllers/static_pages_controller.rb (hunk 1 above)
Comment (sganous) [reply app/controllers/static_pages_controller.rb]: Will fix this up, good catch! Ruby is hard.

**Pull request merged**